homework-submission/main.js

```diff
@@ -0,0 +1,67 @@
+const List = props => {
+  return (
+    <li>
+      {props.description},{props.deadline}
+    </li>
+  );
+};
+
+class TodoList extends React.Component {
+  staticList = [
+    { description: 'Get out of bed', deadline: 'Wed Sep 13 2017' },
+    { description: 'Brush teeth', deadline: 'Thu Sep 14 2017' },
+    { description: 'Eat breakfast', deadline: 'Fri Sep 15 2017' },
+  ];
+
+  dynamicList = [
+    {
+      id: 1,
+      description: 'Get out of bed',
+      deadline: '2017-09-11',
+      done: true,
+    },
+    {
+      id: 2,
+      description: 'Brush teeth',
+      deadline: '2017-09-10',
+      done: false,
+    },
+    {
+      id: 3,
+      description: 'Eat breakfast',
+      deadline: '2017-09-09',
+      done: false,
+    },
+  ];
+
+  render() {
+    return (
+      <div>
+        <h1>static List</h1>
+        <List description={this.staticList[0].description} deadline={this.staticList[0].deadline} />
+        <List description={this.staticList[1].description} deadline={this.staticList[1].deadline} />
+        <List description={this.staticList[2].description} deadline={this.staticList[2].deadline} />
```

Comment:
You could have used this component same as here, instead of a new `li` on line 49 with some adjustments, making it dry code.

Reply:
thank you @Meno-101 ,I hope I have corrected it well